UnitTests/Common/containers/CLookupTable_tests.cpp

```diff
@@ -0,0 +1,101 @@
+/*!
+ * \file CLookupTable_tests.cpp
+ * \brief Unit tests for the lookup table.
+ * \author N. Beishuizen
+ * \version 7.3.1 "Blackbird"
+ *
+ * SU2 Project Website: https://su2code.github.io
+ *
+ * The SU2 Project is maintained by the SU2 Foundation
+ * (http://su2foundation.org)
+ *
+ * Copyright 2012-2022, SU2 Contributors (cf. AUTHORS.md)
+ *
+ * SU2 is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * SU2 is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with SU2. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "catch.hpp"
+#include <sstream>
+#include <filesystem>
+
+#include <dirent.h>
+#include <stdio.h>
+
+#include "../../../Common/include/CConfig.hpp"
+#include "../../../Common/include/containers/CTrapezoidalMap.hpp"
+#include "../../../Common/include/containers/CLookUpTable.hpp"
+#include "../../../Common/include/containers/CFileReaderLUT.hpp"
+
+using namespace std;
```

Comment:
:face_exhaling: just...

Reply:
you don't like STDs? :-)

removed.

Reply:
I like them so much, I prefer typing them everywhere std::string std::vector